src/pc/network/socket/domain_res.c

```diff
@@ -0,0 +1,34 @@
+#include <stdio.h>
+#include "socket.h"
+#include "pc/configfile.h"
+#include "pc/debuglog.h"
+#include "pc/djui/djui.h"
+#ifdef WINSOCK
+#include <winsock2.h>
+#include <ws2tcpip.h>
+#else
+#include <netdb.h>
+#endif
+
+
+void domain_resolution(void) {
+  struct in_addr addr;
+  char *host_name;
+  struct hostent *remoteHost;
+  char* domainname = "";
+  host_name = configJoinIp;
+  if (host_name == NULL) {
+	  return;
+  }
+  int i = 0;
+  remoteHost = gethostbyname(host_name);
```

Comment:
remove host_name variable, just pass `configJoinIp` directly into `gethostbyname()`

Reply:
keep the NULL check, but change it to `configJoinIp`